chore: update description for npm script and baseUrl (#4)
georgbischoff/Tutorials: tutorials/cp-wdi5-cap/cp-wdi5-cap.md

|   Parameter  |   Value  |
| --- | --- |

> |   npm Script  |   use the same script name as we use it to execute the tests locally (`wdi5`).  |

Reviewer: ```suggestion
|   npm Script  |   Use the same script name that is used to execute for local execution of the tests (`wdi5`).  |
```

> |    Base URL  |  Enter the URL of the application against which the tests should be executed. The Application will be available under following URL: `https://<Org Name>-<Space>-bookshop-srv.cfapps.us10-001.hana.ondemand.com/fiori-apps.html#Books-manage`. Depending on your region `us10--001` could also differ.|

Reviewer: ```suggestion
|    Base URL  |  Enter the URL of the application against which the tests shall be executed. The application will be available under the following URL: `https://<Org Name>-<Space>-bookshop-srv.cfapps.us10-001.hana.ondemand.com/fiori-apps.html#Books-manage`. Depending on your region `us10-001` could also differ.|
```
